src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
```

Comment:
- [ ] Following the ducks pattern, your action types should be in the form app/reducer/ACTION_TYPE e.g bookstore/categories/CHECK_STATUS. Please make modification 👍